http-types.cabal

```diff
@@ -35,7 +35,6 @@ Library
                        bytestring >=0.9.1.5 && <0.11,
                        array >=0.2 && <0.6,
                        case-insensitive >=0.2 && <1.3,
-                       blaze-builder >= 0.2.1.4 && < 0.5,
```

Comment:
No, `Builder` is now included in `bytestring` as `Data.ByteString.Builder`. That being said, I do need to update the lower bound on `bytestring`.